addons/pos_self_order/static/src/app/self_order_service.js

```diff
@@ -564,6 +564,8 @@ export class SelfOrder extends Reactive {
                 company: this.company,
                 trackingNumber: order.trackingNumber,
                 bigTrackingNumber: true,
+                pickingService: this.config.self_ordering_service_mode,
+                tableTracker: order.table_stand_number ?? false,
```

Comment:
is there any advantage in explicitly setting this as `false` instead of letting it be `undefined`?